src/test/java/no/unit/nva/cristin/projects/FetchCristinProjectsTest.java

```diff
@@ -213,8 +225,11 @@ void handlerReturnsProjectsWrapperWithAllMetadataButEmptyHitsArrayWhenNoMatchesA
 
     @Test
     void getsCorrectUriWhenCallingQueryProjectsUriBuilder() throws Exception {
-        assertEquals(new URI(QUERY_CRISTIN_PROJECTS_EXAMPLE_URI), cristinApiClientStub
-            .generateQueryProjectsUrl(Map.of(TITLE, RANDOM_TITLE, LANGUAGE_KEY, LANGUAGE_NB)));
+        Map<String, String> params = Map.of(TITLE, RANDOM_TITLE, LANGUAGE_KEY, LANGUAGE_NB, PAGE, PAGE_NUMBER_ONE,
```

Comment:
Can you put a new line on every key-value pair? Also, either use KEY for all parameter keys or do not use it at all